getting-started-sdks.md

```diff
@@ -40,9 +46,10 @@ The following {{site.data.keyword.ibmwatson}} SDKs are supported by {{site.data.
 * [Swift SDK ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://github.com/watson-developer-cloud/swift-sdk){: new_window}
 * [Unity SDK ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://github.com/watson-developer-cloud/unity-sdk){: new_window}
 
-The [API reference ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://{DomainName}/developer/watson/documentation){: new_window} for each service includes information and examples for many of the SDKs, including Java, Node.js, Python, and Ruby.
+The [API reference ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://{DomainName}/apidocs?category=ai){: new_window} for each service includes information and examples for many of the SDKs, including Java, Node.js, Python, Go, and Ruby.
```

Comment:
Do you want to add Swift the list of SDKs, or are you holding off on that one yet?

Reply:
nice catch